src/Compilers/CSharp/Portable/CodeGen/CodeGenerator.cs

```diff
@@ -137,6 +137,11 @@ private bool IsDebugPlus()
             return _module.Compilation.Options.DebugPlusMode;
         }
 
+        private bool IsPEVerifyCompatible()
```

Comment:
Name sits wrong with me. Think it would read better as matching the option name. Say: `EnablePEVerifyCompat`